templates/spin_sidebar.hbs

```diff
@@ -143,6 +143,9 @@
                     Advanced
                 </label>
                 <ul class="menu-list accordion-menu-item-content">
+                    <li><a {{#if (active_content request.spin-path-info "/spin/spin-application-structure" )}} class="active"
+                            {{/if}} href="{{site.info.base_url}}/spin/spin-application-structure">Spin Application Structure</a>
```

Comment:
Sorry for missing this earlier but it occurred to me we can drop "Spin" given this is part of the Spin docs (same in the page title)
```suggestion
                            {{/if}} href="{{site.info.base_url}}/spin/spin-application-structure">Application Structure</a>
```

Reply:
Thanks; I have updated the menu and the page title to be more uniform with what already exists.